docs/Windows.md

```diff
@@ -31,20 +61,39 @@ WARNING: The script 32blit.exe is installed in 'C:\Users\[Name]\AppData\Local\Pr
 
 You will either need to add that to your PATH, or run the tools as `py -m ttblit ...` instead of `32blit ...`.
 
-## Using Visual Studio
-To build using Visual Studio [see here](Windows-VisualStudio.md). The rest of these instructions cover command-line builds.
+You should now be able to run `32blit.exe`, you can test your connection to your 32blit with:
 
-## Building & Running on 32Blit
+```
+32blit flash list
+```
+
+## Building & Running
+
+### On 32Blit
+
+If you want to run code on 32Blit, you should now refer to [Building & Running On 32Blit](32blit.md). On Windows you will need to add `-G"NMake Makefiles"` to your cmake commands, for example:
 
-If you want to run code on 32Blit, you should now refer to [Building & Running On 32Blit](32blit.md). You will need to add `-G"NMake Makefiles"` to your cmake commands and use `nmake` instead of `make`. For example:
 ```
 mkdir build.stm32
 cd build.stm32
 cmake -G"NMake Makefiles" -DCMAKE_TOOLCHAIN_FILE=../32blit.toolchain ..
```

Comment:
The CMake we get from VS should be new enough for
`cmake -B build.stm32 -G"NMake Makefiles" -DCMAKE_TOOLCHAIN_FILE=./32blit.toolchain`
instead of this block.

(and then `cmake --build build.stm32 [...]`)

... but I guess If we're doing that we should update other places too...